.github/workflows/ci.yaml

```diff
@@ -34,6 +34,45 @@ jobs:
       - run: npm ci
       - run: npm run compile-test
       - run: npm test
+  scarb-pick-test:
```

Comment:
Also, I think we're more interested in testing for CairoLS lookup rather than Scarb only. Scarb lookup is more like a detail of the boarder process. Please rename things appropriately.